openpype/hosts/max/plugins/publish/collect_review.py

```diff
@@ -0,0 +1,93 @@
+    label = "Collect Review Data"
+    hosts = ['max']
+    families = ["review"]
+
+    def process(self, instance):
+        nodes = instance.data["members"]
+        focal_length = None
+        camera = None
+        for node in nodes:
+            if rt.classOf(node) in rt.Camera.classes:
+                rt.viewport.setCamera(node)
+                camera = node.name
+                focal_length = node.fov
+
+        attr_values = self.get_attr_values_from_data(instance.data)
+        data = {
+            "review_camera": camera,
+            "frameStart": instance.context.data["frameStart"],
+            "frameEnd": instance.context.data["frameEnd"],
+            "fps": instance.context.data["fps"],
+            "dspGeometry": attr_values.get("dspGeometry"),
+            "dspShapes": attr_values.get("dspShapes"),
+            "dspLights": attr_values.get("dspLights"),
+            "dspCameras": attr_values.get("dspCameras"),
+            "dspHelpers": attr_values.get("dspHelpers"),
+            "dspParticles": attr_values.get("dspParticles"),
+            "dspBones": attr_values.get("dspBones"),
+            "dspBkg": attr_values.get("dspBkg"),
+            "dspGrid": attr_values.get("dspGrid"),
+            "dspSafeFrame": attr_values.get("dspSafeFrame"),
+            "dspFrameNums": attr_values.get("dspFrameNums")
+        }
+        # Enable ftrack functionality
+        instance.data.setdefault("families", []).append('ftrack')
+
+        burnin_members = instance.data.setdefault("burninDataMembers", {})
+        burnin_members["focalLength"] = focal_length
+
+        self.log.debug(f"data:{data}")
+        instance.data.update(data)
+
+    @classmethod
+    def get_attribute_defs(cls):
+
+        return [
+            BoolDef("dspGeometry",
+                    label="Geometry",
+                    default=True),
+            BoolDef("dspShapes",
+                    label="Shapes",
+                    default=False),
+            BoolDef("dspLights",
+                    label="Lights",
+                    default=False),
+            BoolDef("dspCameras",
+                    label="Cameras",
+                    default=False),
+            BoolDef("dspHelpers",
+                    label="Helpers",
+                    default=False),
+            BoolDef("dspParticles",
+                    label="Particle Systems",
+                    default=True),
+            BoolDef("dspBones",
+                    label="Bone Objects",
+                    default=False),
+            BoolDef("dspBkg",
+                    label="Background",
+                    default=True),
+            BoolDef("dspGrid",
+                    label="Active Grid",
+                    default=False),
+            BoolDef("dspSafeFrame",
+                    label="Safe Frames",
+                    default=False),
+            BoolDef("dspFrameNums",
+                    label="Frame Numbers",
+                    default=False)
+        ]
```

Comment:
I follow the naiming here and also follow how max sets by default.
![image](https://github.com/ynput/OpenPype/assets/64118225/63c42a8c-74bc-48d5-a8e7-c65111cf4ccb)

Reply:
I understand - however the Publisher UI doesn't group them under a "Display in Preview" header, or does it? :)

Reply:
> I understand - however the Publisher UI doesn't group them under a "Display in Preview" header, or does it? :)

i am trying to find a way to do this similar to adding maya aov channels in OP settings for publisher. But it seems that I can only do it in OP settings. Maybe I should put it in OP settings instead of exposing it in the code. (It's indeed a long piece of code)

Reply:
I think it's fine to include them and revise after some time in production. Everytime we make an assumption we get humbled but various production needs anyways. In maya, there are very limited options, but that comes from extensive feedback already. In max, enything goes I think untill proven otherwise.